### DISCUSSION TITLE
ios build failed with codemagic while it is working locally

### DISCUSSION THREAD

**Discussion by pranaynemavrize** [codemagic-ci-cd/codemagic-docs#1494] [Q&A] [answered]
Hey i am using codemagic to build React Native applications.

android is working fine but ios build failed with codemagic while it is working locally .

I am not getting any error specific its like - 

Step 8 script `Configure ios build` exited with status code 1

commands - 
`yarn react-native bundle --dev false --entry-file index.js --bundle-output ios/main.jsbundle --platform ios
            cd ios
            gem install cocoapods
            mkdir dist
            pod install`

Build id - 64145a23c230bda99fd731e6

I am facing this issue suddenly 2 week ago before that everything is fine.

can anyone please help ?

**Comment by himesh-cm** [comment, marked as answer]
Hi @pranaynemavrize, Let me check, i will get back to you.

**Reply by pranaynemavrize** [reply]
Waiting for your response and let me know if any other info is required.
Thanks

**Reply by himesh-cm** [reply]
Hi @pranaynemavrize 
Can you please last the successful commit id again, `f9ddba9`, for debugging purposes?

**Reply by pranaynemavrize** [reply]
sure

**Reply by pranaynemavrize** [reply]
@himesh-cm  its failed at same place 

here is some more logs details - 

Successfully installed ruby-macho-2.5.1
Successfully installed cocoapods-downloader-1.6.3
Successfully installed concurrent-ruby-1.2.2
Successfully installed cocoapods-core-1.12.0
Successfully installed cocoapods-1.12.0
Parsing documentation for ruby-macho-2.5.1
Installing ri documentation for ruby-macho-2.5.1
Parsing documentation for cocoapods-downloader-1.6.3
Installing ri documentation for cocoapods-downloader-1.6.3
Parsing documentation for concurrent-ruby-1.2.2
Installing ri documentation for concurrent-ruby-1.2.2
Parsing documentation for cocoapods-core-1.12.0
Installing ri documentation for cocoapods-core-1.12.0
Parsing documentation for cocoapods-1.12.0
Installing ri documentation for cocoapods-1.12.0
Done installing documentation for ruby-macho, cocoapods-downloader, concurrent-ruby, cocoapods-core, cocoapods after 9 seconds
5 gems installed
/Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:184:in `current_thread_backend': uninitialized constant ActiveSupport::XmlMini::IsolatedExecutionState (NameError)
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:103:in `backend='
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:201:in `<module:ActiveSupport>'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:11:in `<top (required)>'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-7.0.1/lib/active_support/core_ext/array/conversions.rb:3:in `<top (required)>'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/cocoapods-1.12.0/lib/cocoapods.rb:9:in `<top (required)>'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/site_ruby/2.7.0/rubygems/core_ext/kernel_require.rb:85:in `require'
	from /Users/builder/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/cocoapods-1.12.0/bin/pod:36:in `<top (required)>'
	from /Users/builder/.rbenv/versions/2.7.2/bin/pod:25:in `load'
	from /Users/builder/.rbenv/versions/2.7.2/bin/pod:25:in `<main>'


Build failed :|
Step 8 script `Configure ios build` exited with status code 1

**Reply by himesh-cm** [reply]
What ruby version do you use locally?

**Reply by pranaynemavrize** [reply]
@himesh-cm 
ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin22]

**Reply by himesh-cm** [reply]
Can you please try the same ruby version on codemagic?
let us know, how it goes

**Reply by pranaynemavrize** [reply]
@himesh-cm 
I aleady tried getting another error 

Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.mobileprovision
Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.provisionprofile

List available code signing certificates in keychain /Users/builder/Library/codemagic-cli-tools/

Searching for files matching /Users/builder/clone/**/*.xcodeproj
Failed to set code signing settings for ios/TSC.xcodeproj
Ruby gem "xcodeproj" is required to configure code signing settings
Install it with [sudo] gem install xcodeproj

**Reply by pranaynemavrize** [reply]
@himesh-cm  any update ?

**Reply by himesh-cm** [reply]
Hi @pranaynemavrize, can you execute `pod install` right before setting up signing: `xcode-project use-profiles` and re-run the build.

**Reply by pranaynemavrize** [reply]
Hey @himesh-cm  Thanks it worked.  😇
